### PR TITLE
Integration testing for project management

### PR DESCRIPTION
**Pull request by tanmoysrt**
#### Describe the changes you have made in this PR -

image

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

File: spec/system/project_spec.rb [hunk 1]
```diff
@@ -0,0 +1,160 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe "Project", type: :system do
+  before do
+    @author = FactoryBot.create(:user)
```
Comment (smritigarg) [inline spec/system/project_spec.rb:7]: **Suggestion** Just like other MRs should we replace the instance variable usage with `let` statements?

File: spec/system/project_spec.rb [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+      visit user_project_path(@public_project.author, @public_project)
+      click_on "Edit"
+      select "Limited access", from: "Project access type"
+      click_on "Update Project"
+      expect(page).to have_text("Limited access")
+    end
+
+    it "add collaborator" do
+      @new_collaborator = FactoryBot.create(:user)
+      visit user_project_path(@private_project.author, @private_project)
```
Comment (smritigarg) [inline spec/system/project_spec.rb:146]: **Suggestion** Shall we put this statement under a subject?

**Review by smritigarg — COMMENTED** [review]
Thanks for the change @tanmoysrt A few suggestions for your consideration, otherwise looks perfectly fine 😄

**Comment by tanmoysrt** [comment]
@smritigarg updated the PR accordingly the suggestion. Please check out

**Pull request merged**